Source/Request.swift

```diff
@@ -178,62 +162,18 @@ open class Request {
         return self
     }
 
-    /// Returns a base64 encoded basic authentication credential as an authorization header dictionary.
+    /// Returns a base64 encoded basic authentication credential as an authorization header tuple.
     ///
     /// - parameter user:     The user.
     /// - parameter password: The password.
     ///
-    /// - returns: A dictionary with Authorization key and credential value or empty dictionary if encoding fails.
-    open static func authorizationHeaderFrom(user: String, password: String) -> [String: String] {
-        guard let data = "\(user):\(password)".data(using: String.Encoding.utf8) else { return [:] }
+    /// - returns: A tuple with Authorization header and credential value if encoding succeeds, `nil` otherwise.
+    open static func authorizationHeaderFrom(user: String, password: String) -> (key: String, value: String)? {
```

Comment:
Not really a part of this PR, but I think the naming here could be better. Perhaps `authorizationHeader(withUser:password)`.